### ISSUE TITLE
pod command with various arguments crashes

### ISSUE THREAD

**Issue by neoneye** [CocoaPods/CocoaPods#5389]
🌈

I just updated to the latest CocoaPods version, because I want to make a framework out of a [git repository](https://github.com/neoneye/SwiftyFORM). However updating to the latest version of CocoaPods has caused the `pod` command to crashing.

* [X] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

I ran `pod lib create Mynewlib` and it crashed.

I ran `pod --version` and it crashed.

## What did you expected to happen?

Running it should complete without crashing.

## What happened instead?

```
PROMPT> mkdir hej
PROMPT> cd hej
PROMPT> pod lib create SwiftyFORM
/Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods/sources_manager.rb:159:in `<module:SourcesManagerMissingConstant>': uninitialized constant Pod::SourcesManagerMissingConstant::Set (NameError)
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods/sources_manager.rb:158:in `<module:Pod>'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods/sources_manager.rb:3:in `<top (required)>'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods/core_overrides.rb:1:in `<top (required)>'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods.rb:71:in `<module:Pod>'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods.rb:17:in `<top (required)>'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/bin/pod:36:in `<top (required)>'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/bin/pod:23:in `load'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/bin/pod:23:in `<main>'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `eval'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `<main>'
PROMPT>
```

## CocoaPods Environment

I just updated cocoapods to the latest stabile version, like this:

```
PROMPT> gem install cocoapods
Fetching: cocoapods-core-1.0.0.gem (100%)
Successfully installed cocoapods-core-1.0.0
Fetching: claide-1.0.0.gem (100%)
Successfully installed claide-1.0.0
Fetching: cocoapods-deintegrate-1.0.0.gem (100%)
Successfully installed cocoapods-deintegrate-1.0.0
Fetching: cocoapods-downloader-1.0.0.gem (100%)
Successfully installed cocoapods-downloader-1.0.0
Fetching: cocoapods-plugins-1.0.0.gem (100%)
Successfully installed cocoapods-plugins-1.0.0
Fetching: cocoapods-search-1.0.0.gem (100%)
Successfully installed cocoapods-search-1.0.0
Fetching: cocoapods-stats-1.0.0.gem (100%)
Successfully installed cocoapods-stats-1.0.0
Fetching: cocoapods-trunk-1.0.0.gem (100%)
Successfully installed cocoapods-trunk-1.0.0
Fetching: cocoapods-try-1.0.0.gem (100%)
Successfully installed cocoapods-try-1.0.0
Fetching: molinillo-0.4.5.gem (100%)
Successfully installed molinillo-0.4.5
Fetching: xcodeproj-1.0.0.gem (100%)
Successfully installed xcodeproj-1.0.0
Fetching: fourflusher-0.3.0.gem (100%)
Successfully installed fourflusher-0.3.0
Fetching: cocoapods-1.0.0.gem (100%)
Successfully installed cocoapods-1.0.0
Parsing documentation for claide-1.0.0
Installing ri documentation for claide-1.0.0
Parsing documentation for cocoapods-1.0.0
Installing ri documentation for cocoapods-1.0.0
Parsing documentation for cocoapods-core-1.0.0
Installing ri documentation for cocoapods-core-1.0.0
Parsing documentation for cocoapods-deintegrate-1.0.0
Installing ri documentation for cocoapods-deintegrate-1.0.0
Parsing documentation for cocoapods-downloader-1.0.0
Installing ri documentation for cocoapods-downloader-1.0.0
Parsing documentation for cocoapods-plugins-1.0.0
Installing ri documentation for cocoapods-plugins-1.0.0
Parsing documentation for cocoapods-search-1.0.0
Installing ri documentation for cocoapods-search-1.0.0
Parsing documentation for cocoapods-stats-1.0.0
Installing ri documentation for cocoapods-stats-1.0.0
Parsing documentation for cocoapods-trunk-1.0.0
Installing ri documentation for cocoapods-trunk-1.0.0
Parsing documentation for cocoapods-try-1.0.0
Installing ri documentation for cocoapods-try-1.0.0
Parsing documentation for fourflusher-0.3.0
Installing ri documentation for fourflusher-0.3.0
Parsing documentation for molinillo-0.4.5
Installing ri documentation for molinillo-0.4.5
Parsing documentation for xcodeproj-1.0.0
Installing ri documentation for xcodeproj-1.0.0
Done installing documentation for claide, cocoapods, cocoapods-core, cocoapods-deintegrate, cocoapods-downloader, cocoapods-plugins, cocoapods-search, cocoapods-stats, cocoapods-trunk, cocoapods-try, fourflusher, molinillo, xcodeproj after 6 seconds
13 gems installed
```

My setup
* OS X 10.11.4 (15E65)
* Xcode 7.3.1 (7D1014)

```
PROMPT> uname -a
Darwin example.local 15.4.0 Darwin Kernel Version 15.4.0: Fri Feb 26 22:08:05 PST 2016; root:xnu-3248.40.184~3/RELEASE_X86_64 x86_64
PROMPT> ruby -v
ruby 2.1.1p76 (2014-02-24 revision 45161) [x86_64-darwin12.0]
PROMPT> rvm -v
rvm 1.25.28 (stable) by Wayne E. Seguin <wayneeseguin@gmail.com>, Michal Papis <mpapis@gmail.com> [https://rvm.io/]
PROMPT> pod --version
/Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods/sources_manager.rb:159:in `<module:SourcesManagerMissingConstant>': uninitialized constant Pod::SourcesManagerMissingConstant::Set (NameError)
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods/sources_manager.rb:158:in `<module:Pod>'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods/sources_manager.rb:3:in `<top (required)>'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods/core_overrides.rb:1:in `<top (required)>'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods.rb:71:in `<module:Pod>'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/lib/cocoapods.rb:17:in `<top (required)>'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/gems/cocoapods-1.0.0/bin/pod:36:in `<top (required)>'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/bin/pod:23:in `load'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/bin/pod:23:in `<main>'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `eval'
	from /Users/johndoe/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `<main>'
PROMPT>
```

**Comment by segiddins** [comment]
Thanks, this has already been fixed on master.

**Comment by drewmclean** [comment]
I just installed 1.0.0 and still get this issue.  Same platform specs as listed above

**Comment by joekh** [comment]
me too with 1.0.0

**Comment by segiddins** [comment]
Yes, the fix was made after 1.0 was released
